.github/maintainers_guide.md

```diff
@@ -42,30 +42,44 @@ When documentation is in a beta state, it requires a new, distinct collection of
-
-4.  (Slack Internal) Communicate the release internally. Include a link to the GitHub Release.
+### Releases
+_For beta releases, see Beta Releases section below:_
+
+Releasing can feel intimidating at first, but rest assured: if you make a mistake, don't fret! npm allows you to unpublish a release within the first 72 hours of publishing (you just won’t be able to use the same version number again). Venture on!
+
+1. Check the status of the package's GitHub Milestone for issues that should be shipped with the release.
+    -  If all issues have been closed, continue with the release.
+    -  If issues are still open, discuss with the team about whether the open issues should be moved to a future release or if the release should be held off until the issues are resolved.
+
+2. Make sure your local `main` branch has the latest changes.
+
+3. Bump the version number in adherence to [Semantic Versioning](http://semver.org/) in `package.json`. (see [Versioning and Tags](https://github.com/slackapi/node-slack-sdk/blob/main/.github/maintainers_guide.md#versioning-and-tags))
+    -  Update any dependency versions in `package.json` and install locally `rm -rf node_modules && npm install`
+    -  Confirm tests pass and code is free of linting errors by running `npm test`.
+    -  Make a single commit with a message, following the format in: ([Example](https://github.com/slackapi/bolt-js/pull/1133/commits/bcc421cd05b50ddcdeb806fcb27a38d7d9f8ede8)).
+    - Create a pull request for the version change ([Example](https://github.com/slackapi/bolt-js/pull/1133))
+    - Add appropriate labels on the PR, including `release`
+4. Once the PR has been approved and tests have passed, merged to main repository.
+    -  Update your local main branch: `git pull origin main`
+    -  Add a version tag (ie, `git tag @slack/bolt@3.6.0`)
+    -  Push the new tag up to origin: `git push --tags origin`
+5. Distribute the release
+    - To publish, you need to be a member of the `slack Org` on npm and set up 2-Factor Auth with your passsword generator of choice. Before you can publish with npm, you must run `npm login` from the command line.
+    - Run `npm publish . --otp YOUR_OTP_CODE`. To generate an OTP (One Time Password), use your password generator of choice (Duo, 1Password)
+6. Close Github Milestone
+    - Close the relevant GitHub Milestone(s) for the release(s)
+    - Move any unfinished, open issues to the next GitHub Milestone
+7. Create GitHub Release(s) with release notes
+    - From the repository, navigate to the **Releases** section and draft a new release
+    - Release notes should mention contributors, issues and PRs ([Example](https://github.com/slackapi/bolt-js/releases/tag/%40slack%2Fbolt%403.6.0))
+
+8. Communicate the release (as appropriate)
+    - **Internal**
+      - Include a brief description and link to the GitHub release
+    - **External**
+      - **Bot Developer Hangout** (`dev4slack.slack.com`) in **#slack-api**
```

Comment:
I don't believe this workspace exists anymore

Reply:
Good catch - I've updated this to include the community slack forum instead!